File: contracts-java/src/main/java/com/review/contract/CarLicenseContract.java
package com.review.contract;

import net.corda.core.contracts.CommandData;
import net.corda.core.contracts.Contract;
import net.corda.core.transactions.LedgerTransaction;


/**
 * A implementation of a basic smart contract in Corda.
 *
 * This contract enforces rules regarding the creation of a valid [CarLicenseState].
 *
 * All contracts must sub-class the [Contract] interface.
 */
public class CarLicenseContract implements Contract{
    public static final String ID = "com.review.contract.CarLicenseContract";


    public interface Commands extends CommandData {

    }

    /**
     * The verify() function of all the states' contracts must not throw an exception for a transaction to be
     * considered valid.
     */

    @Override
    public void verify(LedgerTransaction tx) {

    }
}

File: contracts-kotlin/src/main/kotlin/com/review/contract/CarLicenseContract.kt
package com.review.contract

import com.review.state.CarLicenseState
import net.corda.core.contracts.CommandData
import net.corda.core.contracts.Contract
import net.corda.core.transactions.LedgerTransaction

/**
 * A implementation of a basic smart contract in Corda.
 *
 * This contract enforces rules regarding the creation of a valid [CarLicenseState].
 *
 * All contracts must sub-class the [Contract] interface.
 */

class CarLicenseContract: Contract {

    companion object {
        @JvmStatic
        val ID = "com.review.contract.CarLicenseContract"
    }

    override fun verify(tx: LedgerTransaction) {

    }

    interface Commands: CommandData{
        //Add your new commands here
    }

}